packages/apollos-ui-auth/src/styles.js

```diff
@@ -24,10 +24,9 @@ const FlexedSafeAreaView = styled(
   'ui-auth.FlexedSafeAreaView'
 )(SafeAreaView);
 
-const FlexedKeyboardAvoidingView = styled(
-  { flex: 1 },
-  'ui-auth.FlexedKeyboardAvoidingView'
-)(KeyboardAvoidingView);
+const FlexedKeyboardAvoidingView = styled('ui-auth.FlexedKeyboardAvoidingView')(
```

Comment:
You might need to import `named`
```suggestion
const FlexedKeyboardAvoidingView = named('ui-auth.FlexedKeyboardAvoidingView')(
```

Reply:
Also maybe kill this component and import `KeyboardAvoidingView` directly, since it doesn't actually flex anymore :)

Reply:
Went ahead and removed this component.